getting-started/authenticating-users/app.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+    "fmt"
+    "log"
+    "net/http"
+    "io/ioutil"
+    "os"
+    "encoding/json"
+    "github.com/dgrijalva/jwt-go"
+)
+
+var AUDIENCE string
+var CERTIFICATES = make(map[string]string)
+
+
+func main() {
+    http.HandleFunc("/", indexHandler)
+
+    port := os.Getenv("PORT")
+    if port == "" {
+        port = "8080"
+        log.Printf("Defaulting to port %s", port)
+    }
+
+    log.Printf("Listening on port %s", port)
+    log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
+}
+
+func certs() map[string]string {
+    const url = "https://www.gstatic.com/iap/verify/public_key"
+
+    if len(CERTIFICATES) == 0 {
+        resp, err := http.Get(url)
```

Comment:
Is there a client library to make this request?

Reply:
Please clarify. Isn't http such a client library?